components/Ripple/src/MDCStatefulRippleView.m

```diff
@@ -54,37 +55,45 @@ - (instancetype)initWithCoder:(NSCoder *)coder {
 }
 
 - (void)commonMDCStatefulRippleViewInit {
-  if (_rippleColors == nil) {
```

Comment:
Why does this `nil` check need to change? I would expect both arrays to be `nil` during initialization. If not, please add a separate check for `_rippleColors`.